npm/src/controller/oauth.ts

```diff
@@ -595,6 +595,8 @@ export class OAuthController implements IOAuthController {
     let redirect_uri: string | undefined;
     const { SAMLResponse, idp_hint, RelayState = '' } = body;
 
+    const should_disable_sso_trace = process.env.DISABLE_SSO_TRACE === 'true';
```

Comment:
This should go into the opts, can't use env vars in the npm lib like this. @niwsa Can you show him the right place.